src/drt/src/gc/FlexGC_init.cpp

```diff
@@ -830,8 +926,11 @@ void FlexGCWorker::Impl::init(const frDesign* design)
   // ProfileTask profile("GC:init");
   addNet(design->getTopBlock()->getFakeVSSNet());  //[0] floating VSS
   addNet(design->getTopBlock()->getFakeVDDNet());  //[1] floating VDD
-  initDesign(design);
+  initDesign(design, true);
```

Comment:
Why is skipDR=true here?

Reply:
This init function is only used by the DRWorker and the drc command. In both you shouldn't query dr objects in initDesign because:
- if it's used by FlexDRWorker, then the dr shapes should be handled in initDRWorker
- if it's used by drc command, then we shouldn't assume that there are any dr objects in the regionquery, and we handle the route objects in initNetsFromDesign
In case you are wondering why we need skipDR flag, it's because initDesign is used in initPA which is used in pin access.